src/sentry/integrations/discord/integration.py

```diff
@@ -39,7 +39,37 @@
 
 
 class DiscordIntegration(IntegrationInstallation):
-    pass
+    def get_client(self) -> DiscordClient:
+        org_integration_id = self.org_integration.id if self.org_integration else None
+
+        return DiscordClient(
+            integration_id=self.model.id,  # type:ignore
+            org_integration_id=org_integration_id,
+        )
+
+    def uninstall(self) -> None:
+        # If this is the only org using this Discord server, we should remove
+        # the bot from the server.
+        from sentry.services.hybrid_cloud.integration import integration_service
+
+        installations = integration_service.get_organization_integrations(
+            integration_id=self.model.id,
+            providers=["discord"],
+        )
+
+        # Remove any installations pending deletion
+        active_installations = [i for i in installations if i.status not in (2, 3)]
```

Comment:
Is there an enum here that we can use instead of `i.status not in (2, 3)`? Maybe `ObjectStatus.PENDING_DELETION` and `ObjectStatus.DELETION_IN_PROGRESS`?